docs/source/usage.rst

```diff
@@ -6,8 +6,9 @@ Usage
 Installation
 ------------
 
-
-Under Construction!
+- Available as part of Mantid Workbench
+- From source code with `pip` and conda environment configuration
+- As a conda package: `<https://anaconda.org/neutrons/shiver>`
```

Comment:
This URL isn't rendering as a hyperlink, see https://shiver--107.org.readthedocs.build/en/107/usage.html

Reply:
fixed!